dvc/command/remove.py

```diff
@@ -13,8 +13,8 @@ def run(self):
         for target in self.args.targets:
             try:
                 self.repo.remove(target, outs=self.args.outs)
-            except DvcException:
-                logger.exception(f"failed to remove '{target}'")
+            except DvcException as exc:
+                logger.error(exc)
```

Comment:
Forgot that this hides the `__cause__`. Will update.

Reply:
Done in #5287

Reply:
@skshetry my bad. Thank you